tsrc/cli/review.py

```diff
@@ -1,19 +1,83 @@
 """ Entrypoint for Review """
 
+from operator import itemgetter
+
 from .push import get_token, PushAction
 
+import ui
+
+def get_color_status(status):
+    color = ui.green
+    if status == 'failed':
+        color = ui.red
+    if status == 'pending' or status == 'created':
+        color = ui.blue
+    if status == 'skipped':
+        color = ui.darkgray
+    return color
+
+def get_info_for_merge_status(merge_request):
+    if merge_request['merge_status'] == 'can_be_merged':
+        return (ui.green, 'no conflict', ui.reset)
+    return (ui.red, merge_request['merge_status'], ui.reset)
+def sort_jobs(jobs):
+    return sorted(jobs, key=itemgetter('stage', 'name'))
+
 class ReviewAction(PushAction):
-  def __init__(self, *args, **kwargs):
-      super().__init__(*args, **kwargs)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def display_merge_request(self, merge_request):
+        merge_on_success = merge_request['merge_when_pipeline_succeeds']
+        wip = merge_request['work_in_progress']
+        args = []
```

Comment:
args is  always used for the return value of ArugemntParser.parse() 
prefer `message` or `tokens` for list to be passed to ui.info()

Reply:
done